packages/core/package.json

```diff
@@ -16,6 +16,7 @@
     "codegen": "concurrently \"npm run codegen:btc\" \"npm run codegen:eth\" \"npm run codegen:accounts\"",
     "codegen:btc": "java -jar ./scripts/openapi-generator-cli-4.3.1.jar generate -i https://dev.wallet.henesis.io/docs/v2/btc/v3/api-docs -g typescript-axios -o ./src/__generate__/btc --type-mappings Date=string --skip-validate-spec --config ./codegenConfig.json",
     "codegen:eth": "java -jar ./scripts/openapi-generator-cli-4.3.1.jar generate -i https://dev.wallet.henesis.io/docs/v2/eth/v3/api-docs -g typescript-axios -o ./src/__generate__/eth --type-mappings Date=string --skip-validate-spec --config ./codegenConfig.json",
+    "codegen:fil": "java -jar ./scripts/openapi-generator-cli-4.3.1.jar generate -i http://localhost:8080/v3/api-docs -g typescript-axios -o ./src/__generate__/fil --type-mappings Date=string --skip-validate-spec --config ./codegenConfig.json",
```

Comment:
this code should be removed

Reply:
It should be added later to codegen.
How about to add this code, until dev cluster is setup?

Reply:
Now the target branch is changed to `filecoin` from `develop`, it would be fine.